src/generators/android/android.ts

```diff
@@ -57,7 +57,9 @@ export const android: Generator<
   generatePrimitive: async (client, schema, parentPath) => {
     let type = 'Object';
 
-    if (schema.type === Type.STRING) {
+    if (schema.enum) {
+      type = schema.enum.map((e) => (typeof e === 'string' ? `'${e}'` : `${e}`)).join(' | ');
```

Comment:
This is identical to the JS implementation. Let's double-check it.

Reply:
reverted for Android as of now